### PR TITLE
setObjectID() method slash and space update

### PR DESCRIPTION
**Pull request by gabbygermanson**

### CODE REVIEW

File: src/main/java/com/google/step/MapImage.java [hunk 1]
```diff
@@ -76,10 +76,11 @@ public MapImage(double longitude, double latitude, String cityName, int zoom, in
   }
   
   /** 
-  * Create name based off of attributes: city name, zoom level, month, year.
+  * Create name based off of attributes: year, month, name of city, and zoom level in this order.
   */
   public String setObjectID() {
-      return (cityName + "_" + zoom  + "x_" + month + "_" + year + ".png");
+      String city = city.replaceAll(" ", "_");
```
Comment (thompsond) [inline src/main/java/com/google/step/MapImage.java:82]: Should it be String city = cityName.replaceAll...?

File: src/main/java/com/google/step/MapImage.java [hunk 2]
```diff
@@ -76,10 +76,11 @@ public MapImage(double longitude, double latitude, String cityName, int zoom, in
...
+      return (year + "/" + month  + "/" + city + "/" + zoom + "x.png");
```
Comment (thompsond) [inline src/main/java/com/google/step/MapImage.java:83]: The local variable for objectID is never set here

File: src/main/java/com/google/step/MapImage.java (hunk 1 above)
Comment (gabbygermanson) [reply src/main/java/com/google/step/MapImage.java:82]: Yes thank you

File: src/main/java/com/google/step/MapImage.java (hunk 2 above)
Comment (gabbygermanson) [reply src/main/java/com/google/step/MapImage.java:83]: Thx haha

**Pull request merged**